### ISSUE TITLE
Does this tool works to label object in driving videos.?

### ISSUE THREAD

**Issue by anubhav0fnu** [tzutalin/labelImg#377]
I am using [CityScapes](https://www.cityscapes-dataset.com/login/) & [BDD100K ](https://bair.berkeley.edu/blog/2018/05/30/bdd/) datasets, and currently using two techniques to generate labels(a.k.a Semantic Segmentation Masks images).
1. Using [pre-trained checkpoints](https://github.com/tensorflow/models/blob/master/research/deeplab/g3doc/model_zoo.md),  trained on `xception_**_imagenet` model using ImageNet dataset and later start further training on Cityscapes dataset with learning rate 1e-2. Finally, using the final trained model to label the images.
2. Use the pre-trained model `xception65_cityscapes_trainfine` again [here](https://github.com/tensorflow/models/blob/master/research/deeplab/g3doc/model_zoo.md) to just use it to label images.

So, My question is, will your tool will help me in this task and what would the accuracy of it? Does, it require any kind of training of your existing model? Please suggest.

**Comment by vdalv** [comment]
This tool doesn't support semantic segmentation, and there's no models involved, you have to manually annotate your data by drawing bounding boxes... So no, I don't think it would help you with your task.
